department-of-veterans-affairs/va.gov-cms
# VACMS-3197 aliases part b

## Description
See #3197 
This pr:
1. Forces LC content types to use the patterns created in pr 3204.
2. Unchecks the auto path alias toggle switch after lc node is published.

## Testing done
Behat / visual

## QA steps
1. As an `administrator`,  visit `/node/8296/edit` and visually verify that Url alias widget field reads `/resources-and-support-for-your-va-benefits-and-services`
2. Publish node, then visit `/node/8296/edit` as a `content_editor` and visually verify that Url alias widget not checked.



- [x] As an content_editor change title prior to publish.  Validate that the url changes.
Publish the node
- [x] As an content_editor change title after publish.  Validate that the url does not change.
- [x] As an admin change title and validate that the url does not change.
- [x] As an admin change the alias and validate that the url changes.
- [x]  As and editor edit the title and save the change with a status of draft or ready for review (can not publish).
- [x] as admin or content admin, publish the edit from above.

## Post deployment steps
 - [ ] CD to docroot.
 - [ ] Execute `drush scr ../scripts/content/VACMS-3197-bulk_update_lc_path_aliases-2020-10.php`
**Expected**:
  -  Should take less than a minute to complete the run.
  -  There should be ~95 nodes updated.   
  -  There will be at least 5 failures.
there are at least 5 nodes that will not be updated because they already have paths set.
![image](https://user-images.githubusercontent.com/5752113/97768043-ad7f2500-1af6-11eb-9935-9c6844f98fe8.png)
- [ ] Paste terminal output into comment in this PR.
